src/layouts/PanelLayout/shared/panelLayout.consts.js

```diff
@@ -0,0 +1,22 @@
+import { basesPaths } from './basesPaths.enum';
+
+export const adminLinks = {
+    'department base': basesPaths.DEPARTMENTS_BASE,
```

Comment:
I believe it should be reverted -- string keys with spaces are definitely not the best option, imo